app/controllers/admin/publishers_controller.rb

```diff
@@ -1,19 +1,19 @@
 class Admin::PublishersController < AdminController
   before_action :get_publisher
+  include Search
 
   def index
     @publishers = Publisher
 
     if params[:q].present?
       # Returns an ActiveRecord::Relation of publishers for pagination
-      @publishers = Publisher.where("publishers.id IN (#{sql(params[:q])})").distinct
+      search_query = "%#{remove_prefix_if_necessary(params[:q])}%"
+      @publishers = Publisher.where(search_sql, *[search_query] * 7).distinct
```

Comment:
What does `*[search_query]* 7` mean?

Reply:
Before this there was a SQL injection on the admin controller. I removed that by refactoring the search but didn't realize we could use [placeholder conditions](https://guides.rubyonrails.org/active_record_querying.html#placeholder-conditions) as opposed to `?`. I refactored it so that now it's more clear what's going on 
https://github.com/brave-intl/publishers/pull/1354/commits/9f2060a32ca89c8ea4c8dfaa5c7a8a9e7a39c60b